recipes/yolo/container-group.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "",
+  "readme": "$replace",
+  "container": {
+    "command": [],
+    "environmentVariables": {},
+    "image": "saladtechnologies/yolov8-recipe:1.0.0",
+    "imageCaching": true,
+    "resources": {
+      "cpu": 8,
+      "memory": 16384,
+      "gpuClasses": [
+        "2b73eef8-be49-4667-8fc0-5c0cb127cfe0"
+      ],
+      "storageAmount": 53687091200,
+      "shmSize": 64
+    },
+    "priority": "high"
+  },
+  "autostartPolicy": true,
+  "restartPolicy": "always",
+  "replicas": 3,
+  "readinessProbe": {
+    "failureThreshold": 3,
```

Comment:
Maybe just 1 instead of 3? if the readiness probe doesn't work, it seems unlikely it'll successfully process an inference request a moment later